core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/union/UnionOpDesc.scala

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.texera.workflow.operators.union
+
+import com.google.common.base.Preconditions
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.{OneToOneOpExecConfig, OperatorDescriptor}
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
+
+class UnionOpDesc extends OperatorDescriptor {
+
+  override def operatorExecutor: OpExecConfig = {
+    new OneToOneOpExecConfig(this.operatorIdentifier, _ => new UnionOpExec())
+  }
+
+  override def operatorInfo: OperatorInfo = OperatorInfo(
+    "Union",
+    "unions the output rows from multiple input operators",
+    OperatorGroupConstants.UTILITY_GROUP,
+    1, 1, allowMultiInputs = true
+  )
+
+  override def getOutputSchema(schemas: Array[Schema]): Schema = {
+    Preconditions.checkArgument(schemas.forall(_ == schemas(0)))
+    schemas(0)
+  }
```

Comment:
I am thinking of having a feature where some connections are stopped based on these conditions. Eg: If the two schemas are not same, they can't go into a union operator

Reply:
Agree, nice feature to have.